internal/obsreportconfig/obsreportconfig.go

```diff
@@ -31,6 +31,14 @@ var UseOtelForInternalMetricsfeatureGate = featuregate.GlobalRegistry().MustRegi
 	featuregate.StageAlpha,
 	featuregate.WithRegisterDescription("controls whether the collector uses OpenTelemetry for internal metrics"))
 
+// EnableHighCardinalityMetricsfeatureGate is the feature gate that controls whether the collecor should enable
+// potentially high cardinality metrics.
+var EnableHighCardinalityMetricsfeatureGate = featuregate.GlobalRegistry().MustRegister(
```

Comment:
If this was `DisableHighCardinalityMetricsfeatureGate` instead, we could not have a breaking change immediately, and instead follow the feature gate workflow, where it's off by default on the first release, then on by default w/ the gate still available for anyone that wants it in the next release